monkey/monkey_island/cc/ui/src/components/configuration-components/FormConfig.tsx

```diff
@@ -0,0 +1,43 @@
+import Form from '@rjsf/bootstrap-4';
+import React, {useState} from 'react';
+import _ from 'lodash';
+
+
+export const EXPLOITERS_PATH_PROPAGATION = 'exploitation.exploiters';
+export const EXPLOITERS_CONFIG_PATH = 'propagation.' + EXPLOITERS_PATH_PROPAGATION;
```

Comment:
Are these constants needed here? Let's move them to where they're relevant.

Reply:
Probably just a leftover. I will re/move them as needed. Thanks!